app/src/main/AndroidManifest.xml

```diff
@@ -37,11 +37,18 @@
         </activity>
         <activity
             android:name=".activities.SignUpActivity"
-            android:label="@string/app_name"
+            android:theme="@style/AppTheme.NoActionBar"
             android:windowSoftInputMode="stateHidden|adjustResize"/>
         <activity
             android:name=".activities.LoginActivity"
-            android:windowSoftInputMode="stateHidden|adjustResize"/>
+            android:windowSoftInputMode="stateHidden|adjustResize"
+            android:theme="@style/AppTheme.NoActionBar"/>
+
+            <intent-filter>
```

Comment:
@the-dagger intent filter is supposed to be inside activity tag. this is invalid.